robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -149,12 +149,12 @@ public static class PHEONIX_TUNER {
   }
 
   public static final class SWERVE {
-    public static final double MAX_SPEED_METERS_PER_SECOND = 4.7;
-    public static final double MAX_ANGULAR_RATE_ROTATIONS_PER_SECOND = Math.PI;
+    public static final double MAX_SPEED_METERS_PER_SECOND = 4.023;
+    public static final double MAX_ANGULAR_RATE_ROTATIONS_PER_SECOND = 10.826;
```

Comment:
1. 4.023 mps is only 13.2 fps and our setup will be rated for around 15 fps (4.57 m)
2. When doing drive practice, both Hershey and Kris said they liked `Math.PI * 2` better for max angular velocity since they had more control.